docs/details/dashboards/dashboard-inventory.md

```diff
@@ -13,25 +13,40 @@ Inventory objects form a hierarchy with Node at the top, then Service and Agents
 The **Services** tab displays the individual services, the nodes on which they run, and the Agents that help collect the service metrics along with the following information:
 
 
-**Service name** - The name or identifier associated with the service being monitored.
+| **Column Name**| **Description**|
+|--------------|--------------------------------|
+| Service name|The name or identifier associated with the service being monitored.|                                        
+| Node name | Name or identifier associated with a specific node.| 
+| Monitoring status| The **Monitoring** column summarizes the status of all the Agents assigned to the service.             | 
+|  Address         | The IP address or DNS where the service is currently running. |
+|  Port         | The port number on which the service is running. |
+|  QAN         | You can check Query Analytics information and the Service Overview Dashboard for each service by clicking on the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots"/> icon in the **Options** column.|
+|  Additional information | You can also check additional information about the service, by clicking on the <image src="../../_images/arrow-downward.ico" width="15px" aria-label="downward arrow"/> icon. This expands the service entry to show reference information like service labels and IDs. |
```

Comment:
I understand, this column is called 'Options', right?

Reply:
Fixed.